crates/ruma-macros/src/api/api_metadata.rs

```diff
@@ -110,6 +115,9 @@ impl Parse for Metadata {
                 FieldValue::Added(v) => set_field(&mut added, v)?,
                 FieldValue::Deprecated(v) => set_field(&mut deprecated, v)?,
                 FieldValue::Removed(v) => set_field(&mut removed, v)?,
+                FieldValue::History(h) => {
+                    // println!("history: `{:#?}`", h);
```

Comment:
Currently this is about as far as the macro value gets in the proc-macro process, this bit will print it for debugging.